protocol.md

```diff
@@ -248,8 +271,9 @@ Server MUST ignore the header.
 This example clarifies the response for an `OPTIONS` request. The version used
 in both, request and response, is `1.0.0` while the Server is also capable of
 handling `0.2.2` and `0.2.1`. Uploads with a total size of up to 1GB are
-allowed and the extensions for [Creation](#creation) and
-[Expiration](#expiration) are enabled.
+allowed and the Server recommends that a single chunk should be smaller than
```

Comment:
I think it's intentional, because it's just a description of an example, not part of the specification.